Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierQualificationAwardingBody.cshtml

```diff
@@ -2,9 +2,17 @@
 @model CO.CDP.OrganisationApp.Pages.Supplier.SupplierQualificationAwardingBodyModel
 @{
     var hasError = ((TagBuilder)Html.ValidationMessageFor(m => m.AwardedByPersonOrBodyName)).HasInnerHtml;
+
+    var backLink = $"/organisation/{Model.Id}/supplier-information/supplier-qualification-summary{(Model.QualificationId.HasValue ? "" : "?selected=true")}";
```

Comment:
Do we need test coverage for the back link logic?

Reply:
We have not written test coverage for bank link anywhere yet.  If we need to we might have to address it separately.